ldm/models/diffusion/ddim.py

```diff
@@ -14,12 +14,12 @@
 
 
 class DDIMSampler(object):
-    def __init__(self, model, schedule='linear', device='', **kwargs):
+    def __init__(self, model, schedule='linear', device=None, **kwargs):
```

Comment:
This line and line 7 have been fixed in the apple-mps branch, where all the logic for determining the torch device are folded up into a shared function.